projects/packages/videopress/src/client/components/video-upload-area/style.module.scss

```diff
@@ -0,0 +1,64 @@
+.wrapper {
+	max-width: 552px;
+	width: 100%;
+	border-radius: var(--jp-border-radius);
+	border: 1px dashed var(--jp-gray-20);
+	background-color: var(--jp-gray-0);
+	padding: 48px 0 64px;
+
+	.button {
```

Comment:
Do not override the button, we should follow its specs. If Figma isn't following it, we should report to Design and keep using the specs.

Context: p1651075756803519/1651067591.500319-slack-C02TQF5VAJD